wsi/drm/drmdisplay.cpp

```diff
@@ -403,8 +434,19 @@ void DrmDisplay::SetDrmModeInfo(const std::vector<drmModeModeInfo> &mode_info) {
 }
 
 void DrmDisplay::SetDisplayAttribute(const drmModeModeInfo &mode_info) {
-  width_ = mode_info.hdisplay;
-  height_ = mode_info.vdisplay;
+  // Default resolution of the display
+  if (!IsCustomResolution()) {
```

Comment:
I am still debating the usage of this and GetCustomResolution() in drmdisplay. This is inherited from PhysicalDisplay and should have access to all the protected variables. I think we can just use the variables directly here

Reply:
Accessing variables of super class in derived class directly instead of defining a method to refer that is something debatable. It makes the code difficult to understand and hard to maintain going forward. I would prefer methods in such scenarios.

Reply:
If you still think otherwise, then we can change to either private variable or access directly.

Reply:
access directly is fine as we do the same for others.

Reply:
Accessed directly now!